dev/mysql/oldest/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
-  mysql-57:
-    container_name: sequelize-mysql-57
+  mysql-oldest:
+    container_name: sequelize-mysql-oldest
     image: mysql:5.7
```

Comment:
Should we specify the patch version here as well? Like we did with mysql-latest

Reply:
I hesitated one which patch version to use:

- latest patch version v5.7.38
- or v5.7.0

Reply:
Let's try out 5.7.0

Reply:
Done